### DISCUSSION TITLE
Powerfactory to Pandapower

### DISCUSSION THREAD

**Discussion by Arjunayodya** [e2nIEE/pandapower#2050] [Q&A]
I was working on converting the Powerfactory model to Pandapower, using Engine Mode.

But they got the following error 

from pandapower.converter.powerfactory import from_pfd
![Capture](https://github.com/e2nIEE/pandapower/assets/65343360/03d081a7-bc3b-46c9-bf6d-4e2ed3af9a53)

Tried pip install numpy -U, also didn't work

I am also confused about providing the project name and destination path inside 'from_pfd(app, prj_name="Your Project Name", path_dst="Save Path")'

**Comment by vogt31337** [comment]
@Arjunayodya , without test data this will be hard to reproduce. Maybe can you provide a dataset and try it with the current version of the converter?

**Comment by KS-HTK** [comment]
For the first part it is as @vogt31337 said, without a reproducible example we can't get much info. Even with the example this StackTrace reads more like an issue in the environment so providing the versions used would most likely be helpful.

To the second question:
- `prj_name=` needs to be the name of the project in powerfactory that is shown when opening a project.
- `path_dst=` is the path AND file_name where the network should be saved. So on linux `/home/[user]/my_network.json` would be fine on windows `C:\Users\[user]\Desktop\my_network.json` (windows requires this to be escaped in python `"C:\\[...]"` or use raw string `r"C:\[...]"`
